### PR TITLE
refactor: rename Filter Query Browser to Filter Explorer and improve consistency

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: src/scripts/compile_svelte.mjs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    bundle: true,
+    outfile: "assets/webview.js",
+    plugins: [sveltePlugin()],
+    minify: true,
+    logLevel: "info",
+});
+
+watch(mainFile).on("change", (file) => {
+  console.log(`${file} changed, rebuilding...`);
+  buildFile();
```
Comment (code-withAshish) [inline src/scripts/compile_svelte.mjs:19]: `buildFile()` function is not defined in the context here.

**Review by code-withAshish — CHANGES_REQUESTED** [review]

File: src/scripts/compile_svelte.mjs (hunk 1 above)
Comment (roj1512) [reply src/scripts/compile_svelte.mjs]: Fixed.

**Review by code-withAshish — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
They are called Filter Queries not Filters. How about Filter Query Explorer or simply Filter Queries?

**Comment by code-withAshish** [comment]
> They are called Filter Queries not Filters. How about Filter Query Explorer or simply Filter Queries?

`Filter Query Explorer` would be consistent with the `Updates Explorer`  I agree

**Comment by roj1512** [comment]
Filter Query Explorer is what to go with.

**Comment by KnorpelSenf** [comment]
> Filter Query Explorer is what to go with.

I am Steffen and I approve of this message.

**Comment by code-withAshish** [comment]
@KnorpelSenf @roj1512 
are you guys humans anymore or your consciousness has been shifted to a telegram bot LOL

**Comment by KnorpelSenf** [comment]
We're definitely bots. 🤖

**Pull request merged**